File: sdk/python/examples/chat_completion.py
from llm_engine import LLMEngineClient, ChatCompletionRequest

client = LLMEngineClient(base_url="http://localhost:8000", api_key="test_key")
req = ChatCompletionRequest(
    model="gpt-4",
    messages=[{"role": "user", "content": "Hello LLM Gateway!"}]
)
print("Requesting chat completion...")
# response = client.create_chat_completion(req)
print("Chat completion example ready.")

File: sdk/go/examples/chat_completion.go
package main

import (
	"fmt"
	"github.com/llm-engine/sdk-go"
)

func main() {
	client := llmengine.NewClient(llmengine.Config{
		BaseURL: "http://localhost:8000",
		APIKey:  "test_key",
	})
	fmt.Printf("Go SDK client initialized: %v\n", client)
}
